spec/restaurant_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'restaurant'
+
+describe Restaurant do
+  let(:restaurant) { Restaurant.new }
```

Comment:
I think you could have made a double here because you initialize the restaurant with a new instance of the menu class. So you could first double that menu, and make the restaurant with a menu double here - I found that the easiest way to do it, but only last night so it isn't in my code either.